sunbeam-python/sunbeam/commands/resize.py

```diff
@@ -85,6 +87,16 @@ def resize(
                     jhelper,
                     deployment.openstack_machines_model,
                 ),
+                TerraformInitStep(cinder_volume_tfhelper),
+                DeployCinderVolumeApplicationStep(
```

Comment:
We'll need to remove internal dependencies such as this, they make life very hard. But DeployCinderVolumeApplicationStep is actually calling the openstack-plan, we need to make sure `TerraformInitStep(openstack_tfhelper)` is called before.

Reply:
Fixed